src/main/java/Task.java

```diff
@@ -0,0 +1,25 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description, Boolean isDone) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "\u2713" : "\u2718"); //return tick or [✗] symbols
```

Comment:
Consider saving the symbols as a constant variable to make the code more easily understood at first glance to avoid possible confusion